build.sbt

```diff
@@ -11,3 +16,8 @@ lazy val `clinvar-ingest` = project
     jadeTablePackage := "org.broadinstitute.monster.clinvar.jadeschema.table",
     jadeStructPackage := "org.broadinstitute.monster.clinvar.jadeschema.struct"
   )
+
+lazy val `clinvar-transformation-pipeline` = project
+  .in(file("transformation"))
+  .enablePlugins(MonsterScioPipelinePlugin)
+  .dependsOn(`clinvar-schema`)
```

Comment:
`dependsOn` sticks the Scala classes generated in `clinvar-schema` on the classpath of `clinvar-transformation-pipeline`, so everything still compiles.

Reply:
that's pretty neat, will think of using this